docx/docx_test.go

```diff
@@ -1,55 +1,104 @@
-package docx_test
+package docx
```

Comment:
yeah. So when you split out your test to the `packagename_test` (in this case `docx_test`) package, you can only test exported functions.

However, there was some edge case here https://github.com/opencontrol/compliance-masonry/pull/161/files#diff-13ce5f19e770df462c7202a0913535e5R80
that I didn't have a fixture for. So i needed access to an internal method so i have to move the test to the `packagename` (in this case `docx`) package.